### PR TITLE
space added after comma (bug 1167088)

### PR DESCRIPTION
**Pull request by atiqueahmedziad**

### CODE REVIEW

File: src/media/css/detail/more-info.styl [hunk 1]
```diff
@@ -25,6 +25,7 @@
     }
     dd {
         display: inline-block;
+        margin-right: 3px;
```
Comment (spasovski) [inline src/media/css/detail/more-info.styl]: I like this but we'll also need to consider RTL. There should be some `+rtl() {}` blocks you can reference.

**Comment by atiqueahmedziad** [comment]
Done :) 
Thanks.

**Comment by atiqueahmedziad** [comment]
@spasovski  r?

**Comment by spasovski** [comment]
r+ thanks

**Pull request merged**